src/query/owner/models/id-params.ts

```diff
@@ -1,10 +1,11 @@
 import { ApiProperty } from '@nestjs/swagger';
 import { IsUUID, IsNotEmpty } from "class-validator";
 
-export class IdParams {
+
+export class OwnerIdParams {
 
   @IsUUID(4)
   @IsNotEmpty()
   @ApiProperty()
-  readonly id: string;
+  readonly id!: string;
```

Comment:
Zoals besproken is deze voor nu tijdelijk optioneel om sneller te kunnen prototypen / testen. Wil je hier nog een comment of TODO neerzetten als reminder dat dit nog moet worden omgezet?

Reply:
Ik snap dit comments niet, @kad-busses . Het veld `id` is toch niet optioneel? Nooit niet? Dit is onze uuidv4 gegenereerde id ...

Reply:
Ik had het met stefan er over dat we tijdelijk ownerId optioneel kunnen maken bij sensoren, zodat het testen wat makkelijker is voor jeroen. Dat is hier gedaan: https://github.com/kadaster-labs/sensrnet-registry-backend/blob/950d09b4329ef3a126add1db1fe4f53978eb1b7e/src/commands/sensor/models/bodies/create-body.ts#L12

Reply:
Nee, dat klopt inderdaad, die is zeker niet optioneel. Deze wijziging zit er nu in om snel(ler) te kunnen testen met de front-end. Dan kunnen we gelijk testen met sensoren aanmaken, zonder dat eerst formulieren nodig zijn voor het aanmaken van owners. Zodra dat staat moet de `!` weer verwijderd worden. 

Maar misschien ook wel beter om het gelijk goed te doen, dan kúnnen we het later ook niet vergeten terug te zetten. En minder verwarrend in de code 😊

Reply:
Ah. Ok. Prima!